const fetchLogin = async () => {
    try {
      const response = await fetch(UrlApi);
      if(!response.ok){
        throw new error('Failed to fetch data');
      }
      const data = await response.json();
      if(!data){
        throw new Error('Data failed to response')
      }
      console.log('data: ', data);
      return data;
    }
    catch(error){
      console.log('Hubo un error en el fetchLogin')
    }
  }